tests/conftest.py

```diff
@@ -2704,3 +2704,20 @@ def ns_resources_and_connections_cleanup():
     request.addfinalizer(ns_resources_and_connections_cleanup)
 
     return _create_ns_resources
+
+
+@pytest.fixture(scope="session", autouse=True)
+def collect_logs_fixture(request):
+    """
+    This fixture collects ocs logs after tier execution and this will allow
+    to see the cluster's status after the execution on all execution status options.
+    """
+    def finalizer():
+        """
+        Tracking both logs separately reduce changes of collision
+        """
+        if not config.RUN['cli_params'].get('deploy'):
```

Comment:
Won't this stop us from collecting logs if the deployment and tier tests are executed in the same command?

Reply:
I'm not sure if it will, this was separated as well [here](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/deployment.py#L130-L131) in the deployment.

Reply:
If we execute one command to deploy and run tier 1 like this: `run-ci --deploy -m tier1 ...` we won't enter this `if` statement because `config.RUN['cli_params'].get('deploy')` will return `True`. Is this the expected behavior?